packages/backend/src/graphql/resolvers/clientKeys.ts

```diff
@@ -101,16 +44,19 @@ export const addKeyToClient: MutationResolvers<ApolloContext>['addKeyToClient']
     ctx
   ): Promise<ResolversTypes['AddKeyToClientMutationResponse']> => {
     try {
-      const clientService = await ctx.container.use('clientService')
-      const client = await clientService.addKeyToClient({
+      const paymentPointerService: PaymentPointerService =
+        await ctx.container.use('paymentPointerService')
```

Comment:
```suggestion
      const paymentPointerService =
        await ctx.container.use('paymentPointerService')
```
assuming it isn't needed here?